docs/streams/upgrade-guide.html

```diff
@@ -134,11 +134,44 @@ <h3 class="anchor-heading"><a id="streams_notable_changes" class="anchor-link"><
     </p>
 
     <h4><a id="streams_api_changes_380" href="#streams_api_changes_380">Streams API changes in 3.8.0</a></h3>
+
+    <p>
+        Kafka Streams now supports customizable task assignment strategies via the `task.assignor.class` configuration.
+        The configuration can be set to the fully qualified class name of a custom task assignor implementation
+        that has to extend the new `org.apache.kafka.streams.processor.assignment.TaskAssignor` interface.
```

Comment:
Did we update the "Developer Guide" section about configs accordingly?

Reply:
Nope, that was next on the checklist. But I can do it all in one shot